client/index.html

```diff
@@ -1,13 +1,19 @@
 <!DOCTYPE html>
-<html>
+<html ng-app="myApp">
   <head>
     <meta charset="utf-8">
     <title>TimeShare</title>
+    <script src="app/bower_components/angular/angular.min.js"></script>
+    <script src="app/bower_components/angular-ui-router/release/angular-ui-router.min.js"></script>
   </head>
   <body>
     <button type="button" name="button" onclick="fbLogin()">FB Login</button>
     <script src="https://cdn.firebase.com/js/client/2.2.7/firebase.js"></script>
 
+    <script>
+    var myApp = angular.module('myApp', ['ui.router']);
+    </script>
```

Comment:
this functionality should be included into the app.js file and removed from the index.html